manifests/r/RIA/DigiDoc4/4.2.8.89/RIA.DigiDoc4.locale.en-US.yaml

```
@@ -0,0 +1,25 @@
+# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: RIA.DigiDoc4
+PackageVersion: 4.2.8.89
+PackageLocale: en-US
+Publisher: Riigi Infosüsteemi Amet
+PublisherUrl: https://www.ria.ee/
+PublisherSupportUrl: https://www.id.ee/en/id-help/
+#PrivacyUrl: 
+#Author: 
```

```suggestion
Publisher: RIA
PublisherUrl: https://www.ria.ee/
PublisherSupportUrl: https://www.id.ee/en/id-help/
#PrivacyUrl: 
Author: Riigi Infosüsteemi Amet
```
Seems like they use the short name in the Publisher field in the installer